test/api/v3_apns_test.exs

```diff
@@ -1,15 +1,29 @@
 defmodule MongoosePush.API.V3APNSTest do
   use ExUnit.Case, async: false
   alias MongoosePush.Support.API, as: Tools
-  doctest MongoosePush.API.V3
 
   @url "/v3/notification/f534534543"
 
+  setup_all do
+    if Mix.env() == :integration do
+      HTTPoison.start()
```

Comment:
Why isn't it starting? This seems hacky - it should be there already, and if it's not, then something may be wrong and needs fixing.